clickhouse/provider.go

```diff
@@ -150,26 +151,16 @@ func (p *clickhouseProvider) Configure(ctx context.Context, req provider.Configu
 	// If any of the expected configurations are missing, return
 	// errors with provider-specific guidance.
 
-	if environment == "" {
+	if apiUrl == "" {
 		resp.Diagnostics.AddAttributeError(
-			path.Root("environment"),
-			"Missing ClickHouse OpenAPI Environment",
+			path.Root("api_url"),
+			"Missing ClickHouse OpenAPI API URL",
 			"The provider cannot create the ClickHouse OpenAPI client: missing or empty value for the environment. "+
 				"Set the environment value in the configuration or use the CLICKHOUSE_ENV environment variable. "+
 				"If either is already set, ensure the value is not empty.",
 		)
 	}
```

Comment:
This is likely not to happen due to the logic in the previous lines. Anyway, we should fix error message:

```
	if apiUrl == "" {
		resp.Diagnostics.AddAttributeError(
			path.Root("api_url"),
			"Missing ClickHouse OpenAPI API URL",
			"The provider cannot create the ClickHouse OpenAPI client: missing or empty value for the API url. "+
				"Set the API url value in the configuration or use the CLICKHOUSE_API_URL environment variable. "+
				"If either is already set, ensure the value is not empty.",
		)
	}
	```